tf2onnx/graph.py

```diff
@@ -484,6 +485,20 @@ def inputs(self):
                 all_inputs.append(n)
         return all_inputs
 
+    def make_consts(self, values, np_type=np.int64, skip_conversion=False, raw=True):
+        """create list of consts of same type"""
+        consts = []
+        for value in values:
+            np_val = np.array(value).astype(np_type)
+            key = str(np_val) + "_" + str(np_val.dtype)
+            if key in self._consts:
+                consts.append(self._consts[key])
+            else:
+                const_node = self.make_const(utils.make_name("const"), np_val, skip_conversion, raw)
+                self._consts[key] = const_node.output[0]
+                consts.append(const_node.output[0])
+        return consts
```

Comment:
The logic looks ok, only thing is that the API seems a bit inconsistent. See explanation below:

*   The return type of **make_consts()**  is a list of 'Node.output[0]', which is a list of strings. 
*   Whereas the return type of the function below, **make_const()**, is the Node object itself.

It feels a bit odd that the two similar functions are  returning  very different things (i.e. Node objects versus string lists)

Maybe this function can be refactored to return a list of Nodes, and then the caller can just use a comprehension to extract out the output[0] values.  Thoughts?

Reply:
Then how about we rename the new function like make_const_names or something else, since we are creating it to facilitate the consts creation as much as we could?